Tests/SwiftNavigationTests/TextStateTests.swift

```diff
@@ -1,9 +1,9 @@
-#if canImport(SwiftUI)
```

Comment:
I moved this test from SwiftUINavigation to SwiftNavigation. Currently the only test in here is testing SwiftUI-specific behavior, but still it feels like the test should be in this module.